src/EnergyPlus/WindowManager.cc

```diff
@@ -6097,7 +6097,7 @@ namespace WindowManager {
 			//  ( SurfaceWindow(SurfNum)%FrameEmis*(sigma*(SurfaceWindow(SurfNum)%FrameTempSurfIn+TKelvin)**4 - rmir) + &
 			//    hcin*(SurfaceWindow(SurfNum)%FrameTempSurfIn+TKelvin - tin) )
 
-			FrameHeatGain = SurfaceWindow( SurfNum ).FrameArea * ( 1.0 + SurfaceWindow( SurfNum ).ProjCorrFrIn ) * ( hcin * ( SurfaceWindow( SurfNum ).FrameTempSurfIn + TKelvin - tin ) );
+			FrameHeatGain = SurfaceWindow( SurfNum ).FrameArea * FrameCon * ( SurfaceWindow( SurfNum ).FrameTempSurfOut - SurfaceWindow( SurfNum ).FrameTempSurfIn );
```

Comment:
@EnergyArchmage The defect files from the CR that @mjwitte mentions do indicate that this came about because of the individual load terms not summing to the total. The problem is that this only represents the convection from the frame and not the total heat transfer through the frame (which I was hoping to use for a project, and is implied by some of the report variables that use FrameHeatGain).